src/helpers/geoChart.ts

```diff
@@ -105,3 +105,20 @@ export function isDataOfReasonableSize(
 
     return locationData.length <= limit;
 }
+
+export function calculateAverage(values: number[] = []): number {
+    if (values.length === 0) {
```

Comment:
I think `if (!values.length) {` is enough

Reply:
fixed